### PR TITLE
documentation updates assignment

### PR DESCRIPTION
**Pull request by nmadruga**
Basically adding changes done recently on the Assignment work - using `assigned_sections` instead of `reviewQuestionAssignments`

### CODE REVIEW

**Review by CarlosNZ — CHANGES_REQUESTED** [review]
Cool, just a couple of minor suggestions.

I notice that the summary of the GraphQL assignment mutations is no longer correct, but I can update that [on my PR](https://github.com/openmsupply/application-manager-web-app/pull/1135) once you've had a look, if you like?

**Comment by nmadruga** [comment]
> I notice that the summary of the GraphQL assignment mutations is no longer correct, but I can update that [on my PR](https://github.com/openmsupply/application-manager-web-app/pull/1135) once you've had a look, if you like?

Yes. That would be nice. Thank you

File: docs/internal/Actions-on-Applications.md [hunk 1]
```diff
@@ -39,10 +39,10 @@ Actions more specific to the Consolidator user are available after there is a re
 
 The options are per section of the application, but will show on the application list as the top most possible one (combining the actions available for all sections in the application):
 
-- `Assign` -> When no reviewer is assigned yet (checks `review_question_assignment`)
-- `Re-assign` -> When reviewer is assigned
+- `Assign` -> When no reviewer is assigned yet (to all sections)
+- `Re-assign` -> When reviewer is to each section of the application
```
[the remark this answers is not in the document]
Comment (nmadruga) [reply docs/internal/Actions-on-Applications.md:43]: > Do you mean "when there is a reviewer already assigned to each section"

Yes, that's what I meant. I'll change to that if that's clearer

[the remark this answers is not in the document]
Comment (CarlosNZ) [reply docs/internal/Actions-on-Applications.md]: Okay, I think you must have just missed out a word then:
"When reviewer is _assigned_ to each section of the application"

File: docs/internal/Actions-on-Applications.md [hunk 2]
```diff
@@ -58,11 +58,20 @@ The GraphQL query (`getApplication` and `getApplicationList`) simply gets applic
...
 
-### Review and Assigner home page (or list)
+### Review home page (or list)
 
 On server side we calculate the count of each of the following fields (used to define each available action on the front-end) based on status of `review_assignment` and `review`.
-The GraphQL queries (`getReviewInfo` and `getApplicationList`) will fetch ALL `review_assignment` related to the current application `serial`, for all reviewer `user` of any `level` and `stage`.
-For each existing `review_assignment` with submitted `review` which isn't assigned to the current user, there is the default action to `View`.
+The GraphQL queries (`getReviewInfo` and `getApplicationList`) will fetch ALL `review_assignment` related to the current application `serial`, for all reviewer `user` of the selected `stage` and `level` (or all levels if that's selected).s
+
+On most recent changes the **Assignment tab** was changed to only display one level at each time - and only for the current stage. That was required for simplification of the results displayed and to facilitate on the assignment/re-assignment process. The **Submit** button was also included to validate the assignment/re-assignment.
```
[the remark this answers is not in the document]
Comment (nmadruga) [reply docs/internal/Actions-on-Applications.md:66]: Yes. Good point. Changing now

[the remark this answers is not in the document]
Comment (nmadruga) [reply docs/internal/Actions-on-Applications.md]: I changed the order of these sentences. Hope it makes more sense now.

**Review by CarlosNZ — APPROVED** [review]

**Pull request merged**